lib/resources/postgres_conf.rb

```diff
@@ -93,7 +93,8 @@ def read_content
     def include_files(params)
       include_files = params['include'] || []
       include_files += params['include_if_exists'] || []
-      dirs = params['include_dir'] || []
+      dirs = []
+      dirs.push(params['include_dir']) unless params['include_dir'].nil?
```

Comment:
Another option is to leave the original `||` syntax and do this:

```ruby
dirs = Array(params['include_dir']) || []
```

If `params['include_dir']` is an array, it will leave it. If it's a string, it will become a single-element array. If it's nil, we get an empty array.